boards/xtensa/esp32s2_saola/esp32s2_saola.dts

```diff
@@ -6,7 +6,7 @@
 
 /dts-v1/;
 
-#include <espressif/esp32s2/esp32s2_wrover_n8r2.dtsi>
+#include <espressif/esp32s2/esp32s2_wrover_n4r2.dtsi>
```

Comment:
Is it a fixup of previous commit? Which SiP is the right one?

Reply:
Fixup according to the https://docs.espressif.com/projects/esp-idf/en/latest/esp32s2/hw-reference/esp32s2/user-guide-saola-1-v1.2.html#description-of-components

Reply:
But you first modify this file to:
```
#include <espressif/esp32s2/esp32s2_wrover_n8r2.dtsi>
``` 
in `dts: xtensa: esp32xx rework soc/sip list`, and then to:
```
#include <espressif/esp32s2/esp32s2_wrover_n4r2.dtsi>
```
in `soc: xtensa,riscv: esp32xx: refactor folder structure`.

So I guess everything should be part of  `dts: xtensa: esp32xx rework soc/sip list`, right?